examples/connext_dds/dynamic_data_access_union_discriminator/c++03/CMakeLists.txt

```diff
@@ -33,7 +33,9 @@ set_target_properties(dynamic_data_union_example_cxx2
     PROPERTIES
         OUTPUT_NAME "dynamic_data_union_example")
 
-if(UNIX AND CMAKE_SYSTEM_NAME MATCHES "Linux")
+if(UNIX AND CMAKE_SYSTEM_NAME MATCHES "Linux" 
```

Comment:
Do you need UNIX if it's already "Linux"? Maybe you could simplify this.

Reply:
That's true.